apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -271,7 +272,22 @@ export const include: Includeable[] = [
     ],
   },
   { model: Case, as: 'childCase' },
-  { model: Defendant, as: 'defendants' },
+  {
+    model: Defendant,
+    as: 'defendants',
+    required: false,
+    order: [['created', 'ASC']],
+    include: [
+      {
+        model: Subpoena,
+        as: 'subpoenas',
+        required: false,
+        order: [['created', 'DESC']],
```

Comment:
Should this be ordered in ASC instead? The latest subpoena first. That would at least be good for displaying the status, so that the latest status is shown first.

Reply:
DESC will order in descending order, i.e. latest first, so this is correct.